### DISCUSSION TITLE
Increasing Approach Rate in Difficulty Adjust can turn proper stacks into perfect stacks.

### DISCUSSION THREAD

**Discussion by BdBrian** [ppy/osu#27157] [Ideas] [answered]
Issue is as the title says. This may be a bug, but since it could be intended, I decided to add it as a discussion to play it safe.

With a 'proper stack' I mean a stack detected by the editor, which places the notes in a little staircase.
With a 'perfect stack' I mean the notes perfectly on top of each other.

As an example, on this map, setting the Approach Rate to 10.3 or higher will cause all 1/4 rhythms to become perfect stacks, therefore changing the original beatmap and causing readability issues not intended by the mapper.

https://osu.ppy.sh/beatmapsets/788847#osu/2244390
00:37:411 (6,7,8) - the first triplet
screenshots from editor and autoplay (AR10.3) for comparison
![image](https://github.com/ppy/osu/assets/132909733/3eb8620e-2649-410f-b07e-7912815b90bc)
![image](https://github.com/ppy/osu/assets/132909733/660fba6f-39b1-441a-961f-88e79e1f5c91)

**Comment by cihe13375** [comment, marked as answer]
https://github.com/ppy/osu/discussions/13144
